src/Apps/ArtQuiz/Routes/ArtQuizArtworks.tsx

```diff
@@ -298,7 +303,9 @@ export const ArtQuizArtworks: FC<ArtQuizArtworksProps> = ({ me }) => {
         </Flex>
 
         <Tooltip
-          content="Like it? Hit the heart. Not for you? Choose X."
+          content={tooltipText.map(text => (
+            <Text>{text}</Text>
```

Comment:
You wouldn't want to alter the styling of the tooltip; it's currently `variant="xs"` though why not just adjust the width of the tooltip?

Reply:
I realized that there's an issue adjusting the width of the tooltip. The bottom line `Not for you? Choose X.` is longer than the top. Splitting into 2 elements avoids this.

Reply:
added correct styling